### DISCUSSION TITLE
How can I provide feedback on README/docs/examples?

### DISCUSSION THREAD

**Discussion by eladb** [winglang/wing#499] [General]
This repository has special draft pull request that includes all the contents from `main`. It exists simply as a convenient way to provide feedback on content that is already merged, and not intended to be merged.

For example, see [this comment](https://github.com/winglang/wing/pull/497/files#r1017095563) on our README.

See [#497](https://github.com/winglang/wing/pull/497) for details.

**Comment by ekeren** [comment]
> [eladb](https://github.com/eladb) wants to merge 316 commits into [empty](https://github.com/winglang/wing/tree/empty) from [main](https://github.com/winglang/wing)

Oh, I that's smart!

I've locked the empty branch in the branch protection rule settings
![image](https://user-images.githubusercontent.com/1727147/200773540-76eb87f6-54b6-4298-9692-024d971bdd29.png)

**Reply by eladb** [reply]
NICE!
